### ISSUE TITLE
XBox Controller 17 Buttons stopped working since Steam Update June 1. Steam for Linux

### ISSUE THREAD

**Issue by Nekla** [ValveSoftware/steam-for-linux#5037]
Hello, since the Steam for Linux update June 1. my XBox Controller stopped working. In the Steam for Linux Controller settings the formerly working XBox Controller is replaced by something like Standard X-Input Controller, which is not configurable and doesn`t work in the Steam for Linux Games. Now I need your help to get working my XBox Controller again.
I do not use the built-in standard driver of Ubuntu for the XBox controller, but the Ubuntu Xbox driver from the ppa "sudo apt-add-repository -y ppa: rael-gc / ubuntu-xboxdrv". I need this because I also want to play Windows games from Steam on Ubuntu (Wine). I mean, for example, Assassins Creed II and Assassins Creed Revelations. These need this XBox driver, because otherwise they would not work with the controller. Until the beginning of June, everything with this XBox driver worked great with Steam for Linux. The controller was recognized as an XBox controller until an update for Steam took place. Unfortunately my games for Steam Linux are no longer playable with the controller. This Ubuntu XBox driver provides 17 buttons instead of 15 buttons. Thanks for the help continues.

Regards
Hubert

**Comment by kisak-valve** [comment]
Most likely related to #5036.

**Comment by jcfrosty** [comment]
Yep, same issue here. I'm using Sabayon Linux KDE x64. Everything worked fine until last update. Now the controller works in big picture mode, but doesn't work in any of the games.

**Comment by NoXPhasma** [comment]
My XBOX One S controller doesn't work in any game if I connect it to the Controller API. It works fine in the Big Picture but any game I start up refuses to recognise any input. This is the case wired and wireless.

However, removed from the Controller API the gamepad works perfectly fine, so it's definitely a problem with the Controller API.

https://gist.github.com/NoXPhasma/9ed85e6b0602b656c812acd013ee121f

**Comment by jcfrosty** [comment]
Make sure you don't have a mouse that can be recognized as a gamepad due to
extra buttons. I have a lot of games that have this issue even in Windows.
I have to unplug my mouse, press enter to launch the game while only the
gamepad is connected (or use the gamepad if in Big Picture Mode), and the
game will recognize the gamepad. Then I can plug my mouse back in.

**Comment by NoXPhasma** [comment]
No, this has nothing to do with my mouse. This behaviour didn't exist with my XBOX 360 gamepad on the same system. That one broke four months ago and then I bought the XBOX One S and suddenly I have this behaviour.
